.travis.yml

```diff
@@ -16,22 +16,8 @@ before_script:
   
 jobs:
   include:
-  - stage: minitest
+  - stage: test
     script:
     - mkdir tmp
-    - bundle exec rake minitest
-    
-  - stage: base
-    script:
-    - bundle exec rake build:base:2.5
-  - stage: base
-    script:
-    - bundle exec rake build:base:2.6
-      
-  - stage: spec
-    script:
-    - bundle exec rake spec:2.5
-  - stage: spec
-    script:
-    - bundle exec rake spec:2.6
+    - bundle exec rake test
```

Comment:
This is certainly simpler. :)

I wonder if there's value in running (at least) the minitest tests in multiple Ruby versions. [This](https://docs.travis-ci.com/user/build-stages/#build-stages-and-build-matrix-expansion) suggests that using matrix expansion for a single stage is pretty straightforward. I'll leave it up to you.